### PR TITLE
Alt Pill component style with inverted colors

### PR DESCRIPTION
**Pull request by sezgi**
Companion `styles` PR:
https://github.com/Quartz/styles/pull/37

This is part of the email archive and webview design updates.
https://www.figma.com/file/AbzGRPlLExbQtTpT7STbLr/Email-on-web?node-id=490%3A8969

Screen Shot 2022-03-22 at 7 31 25 PM

Screen Shot 2022-03-22 at 7 31 02 PM

### CODE REVIEW

File: src/components/Pill/Pill.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+type PillProps = {
+  /**
+   * String or node to be rendered inside the pill.
+   */
+  children: React.ReactNode;
+  /**
+   * Icon to display before the children.
+   */
+  icon?: string;
+  pillStyle?: 'default' | 'alt';
```
Comment (adamhooper) [inline src/components/Pill/Pill.tsx:13]: Can we please call these "faint" and "inverted", and make `pillStyle` required?

There's only one usage of `<Pill>` in the wild. The name "default" implies we should respect that one usage ... and I don't think it's worthy of any reverence :).

**Review by adamhooper — CHANGES_REQUESTED** [review]
Looks awesome! One teensy change....

File: src/components/Pill/Pill.tsx (hunk 1 above)
Comment (sezgi) [reply src/components/Pill/Pill.tsx:13]: will do. looks like it's used in `LivingBriefing`. Do you know any examples of this I can look at to test the change?

Screen Shot 2022-03-23 at 9 44 08 AM

Comment (sezgi) [reply src/components/Pill/Pill.tsx:13]: Found it!

Comment (sezgi) [reply src/components/Pill/Pill.tsx]: updated!

**Review by adamhooper — APPROVED** [review]

**Pull request merged**